### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by kamila-patulska**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+}
+
+.text  {
+    text-align: center;
+    color:  rgb(247, 241, 241);
+    position: absolute;
+    top: 50%;
+    left:40%;
+}
+.introduction {
```
Comment (paulus-patulus) [inline css/main.css]: for what is this introduction ? This class name is unclear for me. Maybe you should specify this name by location in html section or by some specific attributes which this class contains

Comment (kamila-patulska) [reply css/main.css]: Ok, but i have to change all names of classes. It's only first draft version.

Comment (paulus-patulus) [reply css/main.css]: Ok, so I will be waiting for your changes.

Comment (kamila-patulska) [reply css/main.css]: So i think it will be the first step of my changes. Love.

**Comment by paulus-patulus** [comment]
Do you need so many similar folders ? - images, gallery, picture ?

**Comment by paulus-patulus** [comment]
BIN +8.28 KB js/loading.gif 
 BIN +1.32 KB js/next.png 
 BIN +1.33 KB js/prev.png 

Images in js folder ? This images are in different places, please clean-up them :)

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+}
+
+#contact-section  {
+    justify-content: center;
+    display: flex;
+    flex-wrap: wrap;
+}
+form  {
+    transition: all 4s ease-in-out;
+}
```
Comment (paulus-patulus) [inline css/main.css]: Maybe move it to .contact-form ?

File: css/main.css [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+.form-control {
+    width: 250px;
+    background: transparent;
+    border: none;
+    outline: none;
+    border-bottom: 0.5px solid gray;
+    color: rgb(46, 44, 44);
+    margin-bottom: 16px;
+}
```
Comment (paulus-patulus) [inline css/main.css:184]: is this different form then .contact-form ?

File: css/main.css [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+    text-align: center; 
+    flex-direction: row;
+    display: inline;
+   
+}
+.materials  {
+    margin-top: 30px;
+    font-size: 20px;
+    font-weight: bold;
+}
```
Comment (paulus-patulus) [inline css/main.css]: for what .materials stand for ?

File: css/main.css [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: 0px;
+    margin-left: 50px;
+    margin-right: 50px;
+    line-height: 20px;
+    font-size: 20px;
+    flex-direction: row;
+    justify-content: space-evenly;
+}
```
Comment (paulus-patulus) [inline css/main.css]: all flex containers in html use this .flex-container class ? Because this name is very general

**Review by paulus-patulus — APPROVED** [review]

**Pull request merged**